library/spit_results.py

```diff
@@ -288,6 +288,46 @@ def classify(key, fact_names, classifications):
     return result
 
 
+JBOSS_EAP_JBOSS_USER = 'jboss.eap.jboss-user'
+
+
+def process_id_u_jboss(fact_names, host_vars):
+    """Process the output from 'id -u jboss', as run by Ansible
+
+    :returns: a dict of key-value pairs to add to the output.
+    """
+
+    # We use the 'id' command to check for jboss because it's been in
+    # GNU coreutils since 1992, so it should be present on every
+    # system we encounter.
+
+    if 'jboss.eap.jboss-user' not in fact_names:
+        return {}
+
+    if 'jboss_eap_id_jboss' not in host_vars:
+        return {JBOSS_EAP_JBOSS_USER:
+                'Error: "id -u jboss" not in Ansible playbook'}
```

Comment:
Not sure this error makes sense. Not really related to a playbook, more that the value wasn't captured on the remote system.

Reply:
I think if the value wasn't captured, it should appear in the output but with special error output.

However, I will change the error.